src/main/java/com/softech/coursesvc/controller/CourseController.java

```diff
@@ -1,17 +1,39 @@
 package com.softech.coursesvc.controller;
 
+import com.softech.coursesvc.CourseList;
+import com.softech.coursesvc.entity.CourseEntity;
+import com.softech.coursesvc.service.CourseSvc;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.List;
+
 @RestController
```

Comment:
Lets add base path as well which all endpoi ts can share in the front part

Reply:
Okay. Will add that too.

Reply:
Something like/coursesvc